tests/e2e/kind/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG IMAGE=kindest/node
-ARG VERSION=1.26
-ARG MINOR=0
+ARG VERSION=1.27
```

Comment:
Just curious why you broke it in version and minor instead of a single version=vMAJOR.MINOR